### ISSUE TITLE
Find new reconstructed PDO data 

### ISSUE THREAD

**Issue by boshek** [boshek/rsoi#35]
@eliocamp 

ATM I am unable to find a reliable data source for reconstructed PDO data. I love the data but it is causing issues for me with CRAN and I need to get this fixed to avoid archiving. 

If this is a feature that you really would like, could you recommend something other than this data source? 

https://www.ncdc.noaa.gov/teleconnections/pdo/data.csv

**Comment by eliocamp** [comment]
Huh. Is the issue that that link is not working? 

That's strange. The root website is working fine. AO data loads well, but PNA took a long in my machine. PDO took ages and then returned an empty table. The ncdc might be having issues.  Maybe send an email to them? https://www.ncdc.noaa.gov/contact 
I can do it, if you don't want to.

**Comment by boshek** [comment]
Great! I've left this too long and am slated to be archived tomorrow so I am just trying to keep the lights on for now.

**Comment by eliocamp** [comment]
Oh, noes! 😱
I'd say to resolve it however you can for now, even if it means scrubbing
the whole function. It's not ideal but it's better than having the package
archived.

**Comment by boshek** [comment]
Hmm.. This looks pretty ugly: https://cran.r-project.org/web/checks/check_results_rsoi.html

**Comment by eliocamp** [comment]
It's not failing on my machine. It must be some intermittent issue with the NOAA servers. 

Maybe there needs to be a more robust method of download that fails gracefully if there is no internet or the url is otherwise unavailable.

**Comment by boshek** [comment]
Right. When they say fail gracefully do they mean catch the error and just return say a message and a NULL? 

Thinking of adding httr as a dependency to abstract that all away.

**Comment by eliocamp** [comment]
Yeah, maybe. Or a 0-row dataframe? 

Although if the data is not available, maybe an error is the best condition. But the thing is, is not an error within the package. How do other packages that rely on online resources work?

**Comment by boshek** [comment]
I think in other packages I've just skipped the test on CRAN and then not run the examples.
